dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6795,6 +6795,21 @@ public String toString() {
                   + "etcd cluster (e.g. http://localhost:2379,http://etcd1:2379)")
           .setScope(Scope.ALL)
           .build();
+  public static final PropertyKey ETCD_USERNAME =
+      stringBuilder(Name.ETCD_USERNAME)
+          .setDescription("User name used for EtcdMembership module to do worker"
+              + " registration. Make sure the given user has the full readwrite"
+              + " role permission on all keys with prefix '/'. "
+              + "Refer to etcd official site for authentication:"
+              + "https://etcd.io/docs/v3.2/op-guide/authentication/")
```

Comment:
nit: this is a sanity check that v3.2 is our target version. Our code doesn't really specify supported etcd version per se? So I'm not sure which version is more likely to happen at our user sites. Maybe another alternative is latest doc link (which changes dynamically)?

Reply:
oh yeah u r right, but i'm not sure which version to put in, let me just say refer to etcd offical doc then.